### PR TITLE
A better UI when Market is closed

### PR DESCRIPTION
**Pull request by binarydevteam**
@brucebinary  display closed icon beside of being disable in markets.

### CODE REVIEW

File: src/components/Icons.jsx [hunk 1]
```diff
@@ -4,6 +4,7 @@ import Add from '../../sass/icons/chart settings/zoom-in/ic-zoomin-light.svg';
 import AddThin from '../../sass/icons/add/ic-add.svg';
 import Candle from '../../sass/icons/chart settings/chart types/candle/ic-candle-normal.svg';
 import Close from '../../sass/icons/close/ic-close.svg';
+import ClosedMarket from '../../sass/icons/close/ic-market-closed-line.svg';
 import Comparison from '../../sass/icons/chart settings/comparison/ic-comparison-normal.svg';
```
Comment (brucebinary) [inline src/components/Icons.jsx]: where is `ic-market-closed-line.svg`? I didn't find it in my source code. Also I think you could just css for this. It's much easier to change later on.

Comment (binarydevteam) [reply src/components/Icons.jsx]: yeah you are right, I forgot to add the SVG file in the push.
I try to follow the prev codes, but I also can attached it as style icon only on css.
which one you prefer ?

**Comment by brucebinary** [comment]
@binarydevteam  Cool it works... I just think that the SVG file should be replaced with a `div` block with CSS. Saves space and easier to customize later. Aside that it should be good to go

**Comment by brucebinary** [comment]
@binarydevteam why did you close this? It's not merged yet

**Comment by brucebinary** [comment]
@binarydevteam since you already split this PR to a few I'm closing this

**Pull request closed without merging**